src/utils/string/StringUtils.js

```diff
@@ -347,7 +347,7 @@ const getChannelListInfo = (channel, selfSignUserId, selfAnonUserId) => {
     channelImage = channel?.channel_image;
     channelName = channel?.name;
     if (!channelName || channelName === '') {
-      channelName = removePrefixTopic(channel?.id);
+      channelName = `#${removePrefixTopic(channel?.id)}`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change is good as it ensures that the channel name always starts with a hashtag. However, there's a potential issue if `channel?.id` is undefined or null. This would result in the channelName being just '#'. It would be better to add a fallback value or error handling for this case.
```diff
- channelName = `#${removePrefixTopic(channel?.id)}`;
+ channelName = channel?.id ? `#${removePrefixTopic(channel?.id)}` : 'fallback_value';
```
This way, if `channel?.id` is not available, the channelName will be set to 'fallback_value' (or any other appropriate default value you choose).


<!-- This is an auto-generated comment by OSS CodeRabbit -->